src/GovUk.Education.ExploreEducationStatistics.Common/Services/FileStorageUtils.cs

```diff
@@ -62,12 +60,9 @@ private enum FileSizeUnit : byte
             };
         }
 
-        public static IDictionary<string, string> GetMetaDataFileMetaValues(string dataFileName)
+        public static IDictionary<string, string> GetMetaDataFileMetaValues()
```

Comment:
Could get rid of this?

Reply:
Good point, I wasn't sure whether to leave this in or not. I've just done this as part of another commit to remove unnecessary wrapper classes `UploadFileOptions`, `UploadStreamOptions` and `UploadTextOptions` used to supply the `BlobClient.metadata` parameter in `BlobStorageService` which cleans things up quite nicely.

Reply:
Cool, wasn't quite expecting those wrappers to be removed. Intention was for them to hold any potential future options to avoid excessive function parameters (even if it's currently a bit sparse with just meta data values). Guess it's not too big a deal with named parameters anyway.

Reply:
Ok I suspected that. If there's more options in future as well as metadata we can put it back. Currently our approach is to even try and get rid of all the metadata properties to rely on the database instead. Now that we have the Subject being created earlier and the public Content database connection there shouldn't be any need for most of them with possibly the exception of the `UserNameKey` field which can become a `CreatedBy` database field of File.